src/AutoBuild.jl

```diff
@@ -161,6 +161,10 @@ function build_tarballs(ARGS, src_name, src_version, sources, script,
         error("Package name \"$(src_name)\" is not a valid identifier")
     end
 
+    if Sys.ARCH != :x86_64
```

Comment:
I'm somewhat confident we can make it work on any platform which can run an x86_64 docker image with relatively little effort, so this error message isn't much useful. I just don't have much need to go through this all the things that need to be changed

Reply:
What about if I change the error to something like: 'not currently supported', or switch it to a warning 'needs further development'? Right now it just errors out with a nonspecific failure; the target audience for this change would be 'new person to Yggdrasil, tries building locally with recent Mac and gets a crash which they can't explain / immediately debug'